app/controllers/reports/regions_controller.rb

```diff
@@ -63,10 +67,17 @@ def details
       [@region, @region.facility_regions].flatten
     end
     @repository = Reports::Repository.new(regions, periods: @period_range)
+    chart_repo = Reports::Repository.new(@region, periods: chart_range)
 
+    ltfu_data = {
+      cumulative_assigned_patients: chart_repo.cumulative_assigned_patients[@region.slug],
+      ltfu_patients: chart_repo.ltfu[@region.slug],
+      ltfu_patients_rate: chart_repo.ltfu_rates[@region.slug],
+      period_info: chart_range.each_with_object({}) { |period, hsh| hsh[period] = period.to_hash }
+    }
```

Comment:
Could pull this into a method. `ltfu_chart_data`?

Reply:
👍